dotnet/samples/KernelSyntaxExamples/Example42_KernelBuilder.cs

```diff
@@ -80,8 +80,7 @@ public static Task RunAsync()
         var templateEngine = new PromptTemplateEngine(logger);
         var kernelConfig = new KernelConfig();
 
-        using var httpHandler = new DefaultHttpRetryHandler(new HttpRetryConfig(), logger);
-        using var httpClient = new HttpClient(httpHandler);
+        using var httpClient = new HttpClient();
```

Comment:
Do we even need to create this client, now?

Reply:
This is for example purposes only when the caller has his own a httpclient implementation that he wants to use in the Kernel instance..

Took the opportunity to add another example using the `OpenAIClient` as well.